redi-connect-front/src/components/organisms/Checklist.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Section, Container, Element, Columns } from 'react-bulma-components'
+import Heading from '../atoms/Heading'
+import { ReactComponent as Check } from '../../assets/images/checkmark.svg'
+
+const checks = {
+  mentor: [
+    'I work in the digital industry',
```

Comment:
i18n? Lets try to have at lease one page (home) completely in the translation file

Reply:
added